pulsar-broker/src/test/java/org/apache/pulsar/client/impl/UnAcknowledgedMessagesTimeoutTest.java

```diff
@@ -322,7 +398,7 @@ public void testFailoverSingleAckedPartitionedTopic() throws Exception {
         assertEquals(messagesReceived, totalMessages);
 
         // 5. Check if Messages redelivered again
-        Thread.sleep(ackTimeOutMillis);
+        Thread.sleep(ackTimeOutMillis * 2);
```

Comment:
Looks like we can remove this line by increasing receive timeout of consumer1?

Reply:
Don't need to remove, verify the timeout of the first round.